src/status_im/utils/transactions.cljs

```diff
@@ -7,7 +7,10 @@
   (case network
     "testnet" "ropsten"
     "testnet_rpc" "ropsten"
-    "mainnet" "api"))
+    "mainnet" "api"
+    "mainnet_rpc"
```

Comment:
should it be "mainnet_rpc" "api" ? btw we can do like this
```
(case network
  ("testnet" "testnet_rpc") "ropsten"
  ("mainnet"  "mainnet_rpc") "api"
  ("rinkeby" "rinkeby_rpc") "rinkeby"
```

Reply:
fixed